sdk/eventhub/azure-eventhubs/setup.py

```diff
@@ -30,7 +30,7 @@
 if not version:
     raise RuntimeError('Cannot find version information')
 
-with open('README.rst') as f:
+with open('README.md') as f:
     readme = f.read()
```

Comment:
If the readme format has changed you will need to change the history.rst as well.
Also you will need to add the content type as markdown to the setup definition.

Reply:
Changed history.rst to .md